src/styles/NavButtons.scss

```diff
@@ -27,6 +27,9 @@ $header-height: 8vh;
     position: relative;
     top: 2px;
   }
+  @media screen and (max-width: 768px) {
```

Comment:
Let's just move these into the above &-text block css. We only want the text to disappear, not the entire button.